### PR TITLE
Add e2e module

### PR DESCRIPTION
**Pull request by farhoud**

### CODE REVIEW

File: e2e/src/fula.test.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import {FileTest, GraphQL, GraphQLSubscription} from "./test-data";
+import wrtc from "wrtc"
+import * as process from "process";
+
+let TIMEOUT
+let serverIds
+
+const beforeRun = ()=>{
+  if(process.env.SERVER_ID){
+    serverIds = process.env.SERVER_ID
```
Comment (gitaaron) [inline e2e/src/fula.test.ts:12]: ahh gotcha - should we rename to BOX_ID (for consistency) and include in README?

File: e2e/src/fula.test.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    TIMEOUT = process.env.TIMEOUT
+  }
+  else {
+    TIMEOUT = 5 * 10000
+  }
+}
+
+
+
+test('Setup', async function (t) {
```
Comment (gitaaron) [inline e2e/src/fula.test.ts:27]: can we rename to 'tester' ? 't' is a little hard to read

**Review by gitaaron — APPROVED** [review]
looks great - feel free to merge when you feel its ready (no need to address my comments)

File: e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Running Test against Box with JS-Fula
+This project use js-fula and tape to run full api test with box.
```
[the remark this answers is not in the document]
Comment (farhoud) [reply e2e/README.md]: yes i will update the doc

File: e2e/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+```shell
+rushx start
+```
+With npm:
+```shell
+npm start
+```
+
+## Roadmap
+- [ ] Add cli to select scenario user want to run
```
[the remark this answers is not in the document]
Comment (farhoud) [reply e2e/README.md]: I will update docs

File: e2e/src/fula.test.ts (hunk 1 above)
Comment (farhoud) [reply e2e/src/fula.test.ts]: Done

**Pull request merged**